xbmc/cores/VideoPlayer/Process/VideoBuffer.h

```diff
@@ -70,10 +70,6 @@ class IVideoBufferPool : public std::enable_shared_from_this<IVideoBufferPool>
   // as soon as it knows parameters: pixFmx, size
   virtual void Configure(AVPixelFormat format, int size) {};
 
-  // configure dimensions of pool. May be called after first buffer is allocated
-  // so renderer knows dimensions of buffer
-  virtual void SetDimensions(int width, int height, int alignedWidth, int alignedHeight) {};
```

Comment:
@popcornmix calling this on the pool makes not too much sense imo. a pool gets configured with a size and as long a pic fits into size the pool does not care anymore for width, height, or strides